packages/core/src/ui/Text.ts

```
@@ -0,0 +1,628 @@
+  }
+
+  @ignoreClone
+  protected override _onTransformChanged(type: number): void {
+    if (type & UITransformModifyFlags.Size || type & UITransformModifyFlags.Pivot) {
+      this._dirtyUpdateFlag |= RendererUpdateFlags.LocalPositionAndBounds;
+    }
+    this._dirtyUpdateFlag |= RendererUpdateFlags.WorldPositionAndBounds;
+  }
+
+  private _isTextNoVisible(): boolean {
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)